doc/source/whatsnew/v1.0.0.rst

```diff
@@ -227,7 +227,7 @@ Other enhancements
 - Added new writer for exporting Stata dta files in version 118, ``StataWriter118``.  This format supports exporting strings containing Unicode characters (:issue:`23573`)
 - :meth:`Series.map` now accepts ``collections.abc.Mapping`` subclasses as a mapper (:issue:`29733`)
 - The ``pandas.datetime`` class is now deprecated. Import from ``datetime`` instead (:issue:`30296`)
-
+- :func:`to_pickle` and :func:`read_pickle` now accept URL (:issue:`30163`)
```

Comment:
`to_pickle` isn't a method. Should be DataFrame.to_pickle.

Reply:
I thought `to_pickle` is intended to be a publicly available function.
I'll update it to `:meth:`DataFrame.to_pickle``.

Reply:
Ah, my apologies, it is a method. I don't believe we document it though (that's a separate issue).

Reply:
No, I should have asked, as I knew it's commonly used as method but the code looks like it's also intended (or allowed) to be used as a function.
Thank you for the clarification. and it's been updated. Thank you!